scripts/utils/options.ts

```diff
@@ -140,11 +140,12 @@ export function getOptions<TOptions extends OptionSpecifier>(
 
       const checkStringValue = (raw: string) => {
         if (option.values && !option.values.includes(raw)) {
-          const possibleOptions = chalk.cyan(option.values.join(', '));
+          const possibleOptions = chalk.cyan(option.values.join('\n'));
```

Comment:
This change seems unrelated to this PR, was it intentional?

Reply:
Unrelated, but intentional. The message was pretty unreadable, so I just quickly improved it slightly